src/components/liquidity_input/index.tsx

```diff
@@ -15,40 +15,64 @@ export enum CurrencyTypes {
 interface LiquidityInputProps {
   currency: CurrencyTypes
   label: string
+  dropdown?: boolean
+  setValue: React.Dispatch<React.SetStateAction<string>>
+  value: string
 }
 
 //TODO need to fix currencies & add icons-dropdown
```

Comment:
we can create the lists of tokens addresses(or download it to the store from anywhere) and use it there for currencies